### PR TITLE
[Improvement] Use Pylint to polish code style

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: mmaction/core/evaluation/eval_hooks.py [hunk 1]
```diff
@@ -40,6 +39,8 @@ def __init__(self, *args, save_best='auto', **kwargs):
 
 if not from_mmcv:
 
+    from mmcv.runner import Hook
```
Comment (congee524) [inline mmaction/core/evaluation/eval_hooks.py:42]: There should be many imported packages that are only used in local code blocks. Do you want to modify them all?

Comment (dreamerlin) [reply mmaction/core/evaluation/eval_hooks.py:42]: Some of them may cause circle import, so just keep it.

File: mmaction/datasets/audio_visual_dataset.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def load_annotations(self):
                     idx += 1
                 # idx for label[s]
                 label = [int(x) for x in line_split[idx:]]
-                assert len(label), f'missing label in line: {line}'
+                assert len(label) != 0, f'missing label in line: {line}'
```
Comment (congee524) [inline mmaction/datasets/audio_visual_dataset.py:68]: maybe `> 0` is better, but it's not a big problem

Comment (dreamerlin) [reply mmaction/datasets/audio_visual_dataset.py:68]: just format according to Pylint which is based on PEP8

**Comment by kennymckormick** [comment]
Is this PR still in WIP? 
image

**Review by kennymckormick — APPROVED** [review]

**Pull request merged**